includes/classes/SiteAutomation/Request.php

```diff
@@ -96,8 +97,26 @@ public function get( $page, $widget, $timeout = 3 ) {
 		 */
 		$bypass_cache = apply_filters( 'sophi_bypass_get_cache', false, $page, $widget );
 
-		if ( ! $bypass_cache ) {
-			$site_automation_data = get_option( "sophi_site_automation_data_{$page}_{$widget}" );
+		if ( ! $bypass_cache && post_type_exists('sophi-response' ) ) {
+			$query = new \WP_Query(
+				[
+					's'                      => "sophi_site_automation_data_{$page}_{$widget}",
```

Comment:
Did you explore using `post_name__in` here instead? Just wondering if someone has ElasticPress and Protected Content enabled (just as an example) if this wouldn't bring the wrong result.

Reply:
(Obviously, this would require an adjustment in the wp_insert_post call below)

Reply:
I think it is a more reliable approach indeed, I have addressed it here https://github.com/globeandmail/sophi-for-wordpress/pull/298/commits/39094a0992f0a117c59ceedb00402d8d9ae31c60 and here https://github.com/globeandmail/sophi-for-wordpress/pull/298/commits/c737ab64c42645f42085e531239d8150c99c44f8